src/main/cpp/feedback/DragonLeds.h

```diff
@@ -59,7 +59,7 @@ class DragonLeds
 	static DragonLeds *m_instance;
 	frc::AddressableLED *m_addressibleLeds;
 	int m_rainbowFirstPixelHue = 0;
-	int m_numberofDiagnosticLEDs = 8;
+	int m_numberofDiagnosticLEDs = 4;
```

Comment:
Have we defined all of the things we need for diagnostics?  4 seems like a small number unless mechgen isn't using any